.github/workflows/release.yml

```
@@ -86,31 +101,129 @@ jobs:
             --entitlements "$APP_NAME/$APP_NAME.entitlements" \
             "$EXPORT_DIR/$APP_NAME.app"
 
-      - name: Package DMG
+      - name: Create ZIP for notarization
         run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2153:info:1:11: Possible misspelling: APP_NAME may not be assigned. Did you mean ZIP_NAME?

(shellcheck)